eosiojava/src/main/java/one/block/eosiojava/models/signatureProvider/EosioTransactionSignatureRequest.java

```diff
@@ -16,6 +16,11 @@
      */
     private String serializedTransaction;
 
+    /**
+     * The serialized (Hex) version of all concatenated context free data
+     */
+    private String contextFreeData;
```

Comment:
Should we name it serializedContextFreeData to distinguish with the un-serialized version?

Reply:
Good call -- updated.